apps/mediator/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:20 AS base
```

Comment:
Since node 20 is under maintenance now, do you think we can upgrade to node:22? I'm not aware of any particular issue with it (or at least no more than the ones we had with 20 :sweat_smile: )

Reply:
Fixed.